main/src/ca/uwaterloo/flix/language/dbg/AstPrinter.scala

```diff
@@ -0,0 +1,55 @@
+package ca.uwaterloo.flix.language.dbg
+
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.language.ast.SourceLocation
+import ca.uwaterloo.flix.language.dbg.Doc.Indent
+import ca.uwaterloo.flix.util.InternalCompilerException
+
+import java.nio.file.{Files, LinkOption, Path}
+
+object AstPrinter {
+
+  /** default indentation width of 4 spaces */
+  private val Indent = 4
+
+  /** default maximum output width of 80 chars */
+  private val Width = 80
```

Comment:
Maybe we should bump it to 120?

Reply:
I think that this is something you'd often want side-by-side with the original code, so small is better IMO. Compromise at 100?

Reply:
Ok